api/src/main/java/com/codeforcommunity/dto/blocks/BlockExport.java

```diff
@@ -0,0 +1,112 @@
+package com.codeforcommunity.dto.blocks;
+
+import java.sql.Timestamp;
+import java.util.List;
+
+public class BlockExport {
+  private String fid;
+  private String status;
+  private Timestamp updatedTimestamp;
+  private String firstName;
+  private String lastName;
+  private String email;
+  private String username;
+  private int userId;
+  private String teamNames;
+
+  /** Constructor for Block & User query. User Team names must be set separately. */
+  public BlockExport(
+      String fid,
+      String status,
+      Timestamp updatedTimestamp,
+      String firstName,
+      String lastName,
+      String email,
+      String username,
+      int userId) {
+    this.fid = fid;
+    this.status = status;
+    this.updatedTimestamp = updatedTimestamp;
+    this.firstName = firstName != null ? firstName : "";
```

Comment:
does Java have a null coalesce operator? 
like `firstName || ""` in JS

Reply:
Nope, this is a fine way to do that